R/base64.R

```diff
@@ -28,3 +28,11 @@
   else
     decodeColNames(x, ...)
 }
+
+interactionSymbol <- "\u2009\u273B\u2009"
+base::Encoding(interactionSymbol) <- "UTF-8"
+
+substituteInteractionSymbol <- function(strings) {
```

Comment:
`prettyInteraction()` or `subInteractionSymbol()`? I'm all for descriptive names, but it feels like this could be a bit more succinct ;)

Reply:
how about `gsubInteractionSym`?

Reply:
Sounds good!